### ISSUE TITLE
Add title attribute to img tags

### ISSUE THREAD

**Issue by onaralili** [onaralili/SplitUp#20]
Many of the image buttons are missing _title_ attribute. In order to improve accessibility, we want to add a short explanatory title attribute to _img_ tags (all of them almost used as a button) such as trash, export etc.
